scripts/mirador.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+echo "Installing Mirador"
+
+#HOME_DIR=$1
+#if [ -f "$HOME_DIR/islandora/configs/variables" ]; then
+#  . "$HOME_DIR"/islandora/configs/variables
+#fi
+#cd "$HOME_DIR"
+
+cd /tmp && { curl -LOk https://github.com/NVLI/mirador-js/archive/master.zip; curl -LOk https://ftp.drupal.org/files/projects/mirador-8.x-1.x-dev.zip ; cd -;}
```

Comment:
I would uncomment lines 4-8, then you can use some pre-set variables from the [variables](https://github.com/Islandora-CLAW/claw_vagrant/blob/master/configs/variables) file.

For instance, I would check if this was downloaded into `$DOWNLOAD_DIR`. If yes, don't re-download. If no, download.